syft/he/paillier/basic.py

```diff
@@ -119,17 +130,85 @@ def sum(self, dim=None):
             op = self.data.sum(axis=dim)
             return PaillierTensor(self.public_key, op, False, fixed_point_conf=self.fixed_point_conf)
 
+
+class Float():
```

Comment:
Can you check this, again? this seems like an extra, I don't see it [here](https://github.com/OpenMined/PySyft/blob/master/syft/he/paillier/basic.py)

Reply:
hmm weird, I thought I had just pulled down from master. Let me take that out.